Shopify create update clarification (#4024)
braze-inc/braze-docs: _docs/_hidden/private_betas/shopify_create_update.md

- Customer creates an account
- Customer email or phone is collected in a Shopify pop-up form

> - Customer email is collected on your store’s via Shopify's footer

Author: ```suggestion
- Customer email is collected on your store from Shopify's footer
```
